.github/workflows/super-linter.yml

```diff
@@ -32,6 +32,7 @@ jobs:
           VALIDATE_OPENAPI: false
           VALIDATE_JSCPD: false
           VALIDATE_GO: false
+          VALIDATE_GO_MODULES: false
```

Comment:
maybe we can rather configure the linter to ignore the known failing modules (this `zondax` thing?)

Reply:
After googling a bit about the linter I think `golangci-lint` plus all the semgrep stuff etc. is just fine

Reply:
I removed the GO_MODULE from the linter also because it was complaining against changes required by the golangci